src/gui/CMakeLists.txt

```diff
@@ -255,6 +260,12 @@ endif()
 
 add_library(nextcloudCore STATIC ${final_src})
 
+if(WITH_WEBENGINE)
+  set(link_webengine "Qt5::WebEngineWidgets")
```

Comment:
no, you can have more than one call to `target_link_libraries` that is much better than adding a new variable that you are forced to set empty when you do not need it